statelyai/eng-blog
# Properly handle SEO in the blog

Closes https://github.com/statelyai/eng-blog/issues/7

This PR adds support for the required meta tags and open graph metadata to the home page and post page.

## How to test
Run the blog locally and see `<meta />` elements under `head` in the devtools -> element panel.
